package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "higlass-pileup",
-  "version": "0.4.0",
+  "version": "0.4.1",
```

Comment:
This should definitely be at least a minor version bump if not a major. We could just go ahead and release v1.0.0 with these changes.

Reply:
Done